config/HELICSConfig.cmake.in

```diff
@@ -50,6 +50,7 @@ find_program(HELICS_BROKER helics_broker
 set(HELICS_HAVE_ZEROMQ @HELICS_HAVE_ZEROMQ@)
 set(HELICS_HAVE_MPI @HELICS_HAVE_MPI@)
 
+set(SAVED_PARENT_CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH})
 set(CMAKE_MODULE_PATH ${PACKAGE_PREFIX_DIR}/@HELICS_CMAKECONFIG_INSTALL_DIR@)
```

Comment:
if you wanted to override it wouldn't this need some sort of FORCE  modifier,  Otherwise wouldn't the set only work if it wasn't in the cache already?

Reply:
`set` works for overriding the value even if the variable is in the cache, the new value just won't persist (which is fine, since the saved value gets restored before leaving HELICSConfig.cmake) -- only reason this didn't seem to affect HELICS-FMI was because finding HELICS happened after most of the other files from its module path have already been included. 

(CMAKE_MODULE_PATH says it is intended to be set by the project, and at least with how we've been setting it, won't be in the cache unless the user sets it).

Reply:
I'd thought about making it append to the existing CMAKE_MODULE_PATH, but then there is the chance that the project looking for HELICS or the HELICS package will use an incompatible variant of a file with the same name.